Makefile

```diff
@@ -34,16 +34,27 @@ test-unit:
 olympus-docker:
 	docker build -t gopackages/olympus -f cmd/olympus/Dockerfile .
 
+
+.PHONY: alldeps
+alldeps:
+	docker-compose -p athensdeps up -d mysql
+	docker-compose -p athensdeps up -d postgres
+	docker-compose -p athensdeps up -d mongo
+	docker-compose -p athensdeps up -d redis
+	docker-compose -p athensdeps up -d minio
+	docker-compose -p athensdeps up -d jaeger
+	echo "sleeping for a bit to wait for the DB to come up"
+	sleep 5	
+
 .PHONY: dev
 dev:
-	docker-compose -p athensdev up -d mysql
-	docker-compose -p athensdev up -d postgres
 	docker-compose -p athensdev up -d mongo
 	docker-compose -p athensdev up -d redis
-	docker-compose -p athensdev up -d minio
-	docker-compose -p athensdev up -d jaeger
-	echo "sleeping for a bit to wait for the DB to come up"
-	sleep 5
+
+.PHONY: down
+down:
+	docker-compose -p athensdev down
```

Comment:
will this tear down `make alldeps`?
maybe we should merge athensdev and athensdep
or create another down for deps